wire-ios/Wire-iOS/Sources/UserInterface/Conversation Options/CreateSecureGuestLink/CreateSecureGuestLinkViewController.swift

```diff
@@ -399,6 +400,21 @@ extension CreateSecureGuestLinkViewController: UITextFieldDelegate {
 
     @objc
     func textFieldDidChange(_ textField: UITextField) {
+        evaluateTextfieldsAndToggleCreateLinkButtonState()
+    }
+
+    func textFieldShouldReturn(_ textField: UITextField) -> Bool {
+        if textField == securedGuestLinkPasswordTextfield {
+            securedGuestLinkPasswordValidatedTextField.becomeFirstResponder()
+        } else {
+            textField.resignFirstResponder()
+            evaluateTextfieldsAndToggleCreateLinkButtonState()
+        }
+
+        return true
```

Comment:
In this function I would consider small comments as helpful to quickly understand:
```swift
// move focus next to enter password 
...
// dismiss keyboard and finalize
...

return true
```

Reply:
Done!